src/hooks/useDetailsStats.ts

```diff
@@ -2,23 +2,25 @@ import { getStatsForDetails } from 'api/stats';
 import { useEntityType } from 'context/EntityContext';
 import { hasLength } from 'utils/misc-utils';
 import { CatalogStats_Details } from 'types';
-import useDetailsUsageStore from 'components/shared/Entity/Details/Usage/useDetailsUsageStore';
 import { useMemo } from 'react';
 import { DateTime, Interval } from 'luxon';
 import { find } from 'lodash';
 import { useQuery } from '@supabase-cache-helpers/postgrest-swr';
+import { LUXON_GRAIN_SETTINGS, defaultQueryDateFormat } from 'services/luxon';
+import { useDetailsUsageStore } from 'stores/DetailsUsage/useDetailsUsageStore';
 
-function useDetailsStats(catalogName: string, grain: string) {
+function useDetailsStats(catalogName: string) {
     const entityType = useEntityType();
-    const range = useDetailsUsageStore((store) => store.range);
+
+    const [range] = useDetailsUsageStore((store) => [store.range]);
```

Comment:
Why is an array returned by the hook if a destructuring assignment is used to set its variable target? Also, the input argument for the selector is called _store_ as opposed to the traditional _state_.

Reply:
So you can add and remove items easier. `Details Usage` never used the `state` approach.